src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Provider } from 'react-redux';
+import AppRoutes from './routes';
+import store from './redux/store';
+import NavBar from './components/navbar/NavBar';
+import './media-queries/Small.css';
+import './media-queries/Large.css';
+import './media-queries/XLarge.css';
+import './media-queries/XXLarge.css';
+import './media-queries/Desktop.css';
+import MobileNavBar from './components/navbar/MobileNavBar';
+
+function App() {
```

Comment:
- [ ] When I run as per instructions `localhost:3000` in my browser, while running the server at 3001, this happens on Chrome:

![image](https://user-images.githubusercontent.com/57421823/216732744-67bd7312-7062-4172-a1ef-8d96305a9b89.png)

However, in Firefox it works, but with similar console errors:

![image](https://user-images.githubusercontent.com/57421823/216732831-a2c25bdf-fb2d-4a2f-a9e1-83897f06baaa.png)

I'll continue the review using Firefox.

Kindly correct this behavior, the shown console errors can be of great help. 👍 

- [ ] I cannot login or sign up into the app, as the buttons provided greet me with no content:


![image](https://user-images.githubusercontent.com/57421823/216733303-515144cb-69cc-416f-8382-0c908ae9b22b.png)

![image](https://user-images.githubusercontent.com/57421823/216733325-e560e3da-c718-40de-ba7d-19f81662d07d.png)

I used "npm install", however, it didn't work:

![image](https://user-images.githubusercontent.com/57421823/216734419-da0179dd-3a1c-437b-81f9-eb6afc97ecb0.png)

I had to use "npm install --force", and it worked.

No requests were made to the server (it didn't move), so I suspect the issue is entirely in the frontend:

![image](https://user-images.githubusercontent.com/57421823/216733551-ebc77341-6792-4815-bbcc-125e72b34513.png)

My .env file with the `rails secret` key, as per instructions:

![image](https://user-images.githubusercontent.com/57421823/216733777-62a8cf65-1715-4be1-a487-705011363c34.png)

No errors seem to take place in the backend.

Regarding these 2 last review comments, if these are bugs, kindly fix them so your app works as intended.  If the app is working for you and there are special procedures to run your app, please put those instructions in your readme file so your users can run your app smoothly.

_note: This means I couldn't review your reservations, design, mobile implementation, add, and delete functionality.  These features will be taken a look in the next review after this issue has been resolved._